poc/rsabssa.py

```diff
@@ -57,7 +58,10 @@ def RSAVP1(public_key: RsaKey, r: int) -> int:
     return pow(r, e, n)
 
 
-def random_integer_uniform(min: int, max: int) -> int:
+def random_integer_uniform(m: int, n: int) -> int:
+    # Implement rejection sampling.
+    # This is for reference only; most cryptographic libraries include
+    # functions to generate random integers from a uniform distribution.
     range = max - min
```

Comment:
This will not work anymore, since you change the names of the parameters

Reply:
Good catch, thank you!